<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowTitle">
   <string>ЛР8 — Система управління тиром (Варіант 2)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1">
    <item>
     <layout class="QHBoxLayout" name="controls">
      <item><widget class="QPushButton" name="btnStart"><property name="text"><string>Старт групи</string></property></widget></item>
      <item><widget class="QPushButton" name="btnStop"><property name="text"><string>Стоп (переривання)</string></property></widget></item>
      <item><widget class="QPushButton" name="btnReplace"><property name="text"><string>Заміна банок (30с)</string></property></widget></item>
      <item><widget class="QPushButton" name="btnReload"><property name="text"><string>Перезарядка (30с)</string></property></widget></item>
      <item><widget class="QPushButton" name="btnNext"><property name="text"><string>Наступна група</string></property></widget></item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="grid">
      <item row="0" column="0">
       <widget class="QGroupBox" name="lane1"><property name="title"><string>Місце 1</string></property>
        <layout class="QVBoxLayout"><item><widget class="QProgressBar" name="pbShots1"><property name="maximum"><number>10</number></property></widget></item><item><widget class="QProgressBar" name="pbTargets1"><property name="maximum"><number>9</number></property></widget></item><item><widget class="QLabel" name="lblState1"><property name="text"><string>Idle</string></property></widget></item></layout>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QGroupBox" name="lane2"><property name="title"><string>Місце 2</string></property>
        <layout class="QVBoxLayout"><item><widget class="QProgressBar" name="pbShots2"><property name="maximum"><number>10</number></property></widget></item><item><widget class="QProgressBar" name="pbTargets2"><property name="maximum"><number>9</number></property></widget></item><item><widget class="QLabel" name="lblState2"><property name="text"><string>Idle</string></property></widget></item></layout>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QGroupBox" name="lane3"><property name="title"><string>Місце 3</string></property>
        <layout class="QVBoxLayout"><item><widget class="QProgressBar" name="pbShots3"><property name="maximum"><number>10</number></property></widget></item><item><widget class="QProgressBar" name="pbTargets3"><property name="maximum"><number>9</number></property></widget></item><item><widget class="QLabel" name="lblState3"><property name="text"><string>Idle</string></property></widget></item></layout>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QGroupBox" name="lane4"><property name="title"><string>Місце 4</string></property>
        <layout class="QVBoxLayout"><item><widget class="QProgressBar" name="pbShots4"><property name="maximum"><number>10</number></property></widget></item><item><widget class="QProgressBar" name="pbTargets4"><property name="maximum"><number>9</number></property></widget></item><item><widget class="QLabel" name="lblState4"><property name="text"><string>Idle</string></property></widget></item></layout>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QGroupBox" name="lane5"><property name="title"><string>Місце 5</string></property>
        <layout class="QVBoxLayout"><item><widget class="QProgressBar" name="pbShots5"><property name="maximum"><number>10</number></property></widget></item><item><widget class="QProgressBar" name="pbTargets5"><property name="maximum"><number>9</number></property></widget></item><item><widget class="QLabel" name="lblState5"><property name="text"><string>Idle</string></property></widget></item></layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
